ShoppingM8-be/src/main/resources/db/changelog/004-product.yaml

```diff
@@ -1,6 +1,6 @@
 databaseChangeLog:
   - changeSet:
-      id: 1
+      id: 4
```

Comment:
Only id is here changed, why?

Reply:
Because changeset ids should be unique and this one was not

Reply:
OK, I thought it was id of this particular changeset (i.e. 004-product could have id x, and after some changes to the changeset id wolud be changed)